#Header header    # ROS Message headaer

int32 error0      # Axis 0 error
uint8 state0      # Axis 0 state
float32 pos0      # Axis 0 encoder position

int32 error1      # Axis 1 error
uint8 state1      # Axis 1 state
float32 pos1      # Axis 1 velocity
